src/renderer/components/NewProjectModal/NewProjectView.tsx

```diff
@@ -0,0 +1,110 @@
+import { styled, Stack, Box, TextField } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import { ChangeEvent, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { makeProjectWithoutMedia } from 'renderer/util';
+import { projectCreated } from 'renderer/store/actions';
+import CancelButton from '../CancelButton';
+import ActionButton from '../ActionButton';
+import colors from '../../colors';
+import { Project } from '../../../sharedTypes';
+import ModalTitle from '../ModalTitle';
+
+interface Props {
+  closeModal: () => void;
+  nextView: () => void;
+}
+
+const CustomStack = styled(Stack)`
+  width: 100%;
+`;
+
+const CustomTextField = styled(TextField)`
+  color: ${colors.white};
+`;
+
+const Container = styled(Box)`
+  background-color: ${colors.grey[700]};
+  height: 15vw;
+`;
+
+const NewProjectView = ({ closeModal, nextView }: Props) => {
+  const [projName, setProjName] = useState<string>('');
+  const [isAwaitingProjectName, setIsAwaitingProjectName] =
+    useState<boolean>(true);
+
+  const dispatch = useDispatch();
+
+  const setProjectInStore = async (project: Project) => {
+    dispatch(projectCreated(project));
+  };
+
+  const handleContinue = async () => {
+    const project = await makeProjectWithoutMedia(projName);
+    if (project === null) {
+      return;
+    }
+    setProjectInStore(project);
+    nextView();
+  };
+
+  const handleProjectNameInput = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setProjName(event.target.value);
+    if (event.target.value !== '') {
+      setIsAwaitingProjectName(false);
+    } else {
+      setIsAwaitingProjectName(true);
+    }
+  };
+
+  const continueButton = (
+    <ActionButton onClick={handleContinue} disabled={isAwaitingProjectName}>
+      Continue
+    </ActionButton>
+  );
+
+  const cancelButton = <CancelButton onClick={closeModal}>Cancel</CancelButton>;
+
+  return (
+    <Container>
+      <CustomStack
+        direction="column"
+        alignItems="flex-start"
+        justifyContent="space-between"
+        sx={{ height: '100%' }}
+      >
+        <CustomStack
+          direction="row"
+          alignItems="center"
+          justifyContent="space-between"
+        >
+          <ModalTitle>New Project</ModalTitle>
+          <Box onClick={closeModal}>
+            <CloseIcon sx={{ color: colors.yellow[500], fontSize: 36 }} />
+          </Box>
+        </CustomStack>
+        <CustomStack>
+          <CustomTextField
```

Comment:
Figuring out how to change the non-focus and input text colours for this is pretty crucial - the current contrast is really hard to see.

Reply:
Turns the color attribute in the TextField component does not take actual colors. Check the docs. https://mui.com/material-ui/api/text-field/

Color just changes the outline of the TextField and not the text color that the user types.